### PR TITLE
chore: set replica of controller as 2

### PR DESCRIPTION
**Pull request by andyzhangx**
**What type of PR is this?**
/kind feature

**What this PR does / why we need it**:
set replica of controller as 2

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Release note**:
```
none
```

### CODE REVIEW

File: charts/latest/azuredisk-csi-driver/values.yaml [hunk 1]
```diff
@@ -39,4 +39,4 @@ rbac:
   create: true
 
 controller:
-  replicas: 3
+  replicas: 2
```
Comment (ZeroMagic) [inline charts/latest/azuredisk-csi-driver/values.yaml]: Currently, we just added the follow rule in `azuredisk-external-provisioner-role`.
```
- apiGroups: ["coordination.k8s.io"]
  resources: ["leases"]
  verbs: ["get", "watch", "list", "delete", "update", "create"]
```
`azuredisk-external-attacher-role`, `azuredisk-external-snapshotter-role` and `azuredisk-external-resizer-role` also need the rule. We can add them in this pr. 
Then we can judge the replica num 2 whether it is suitable.

Comment (ZeroMagic) [reply charts/latest/azuredisk-csi-driver/values.yaml]: This rule is used for leader election

Comment (andyzhangx) [reply charts/latest/azuredisk-csi-driver/values.yaml]: good catch, fixed.

**Comment by andyzhangx** [comment]
/hold
also need to update rbac setting of chart config

**Pull request merged**